### ISSUE TITLE
Moving Servers Summary into the Status bar.

### ISSUE THREAD

**Issue by n3oxmind** [GNS3/gns3-gui#2551]
I modified almost every part of gns3 gui (hide title bar, menu bar, ..., etc) just to get more clear and wide space for my project that extend into a several workspaces the only think that i'm still fighting is that I want a summary of performance (CPU and RAM usage) this feature already provided as Servers Summary which is a docked window. Unfortunately, docked window take a lot of space and its only a summary of performance it will be nice if the Servers Summary could be shown in the TASK BAR instead.  

In short, I'm requesting to move the `Servers Summary` from docked window into the right side of the status bar.

**Comment by grossmj** [comment]
How could we fit a list of servers in the status bar?

**Comment by n3oxmind** [comment]
The status bar will have only two numbers that is the total CPU usage and total RAM usage. see the picture bellow. 
![rhce-kerberos-ldap - gns3_005](https://user-images.githubusercontent.com/10103340/43809002-9b62ab7c-9a64-11e8-9c37-992ae02503d4.png)

one more thing, the Server Summary List does not have a resources usage next to the list items. for me its not useful, I don't know if others are using this for something else.

Thanks,

**Comment by grossmj** [comment]
> The status bar will have only two numbers that is the total CPU usage and total RAM usage

This would work only if you use a local server (not the GNS3 VM or any other remote server).

**Comment by n3oxmind** [comment]
@grossmj, is it possible to add this as a feature (for linux users).
Thank you,

**Comment by saviodsouza** [comment]
What if there are multiple servers for a large topology?

I would suggest create smalll buttons to quick hide and unhide the docks.

**Comment by n3oxmind** [comment]
closing this issue since its not that important.
